toolbox/src/base/CoderBlockInformation.cpp

```diff
@@ -90,8 +90,8 @@ BlockInformation::VectorSize CoderBlockInformation::getOutputPortWidth(const Por
     return pImpl->outputPortDimensions.at(idx).at(1);
 }
 
-wbt::Signal CoderBlockInformation::getInputPortSignal(const PortIndex idx,
-                                                      const VectorSize size) const
+const wbt::Signal CoderBlockInformation::getInputPortSignal(const PortIndex idx,
```

Comment:
I am not sure if changing the constness of a return value of a function changes anything at all, but if it working I do not think it is a big problem.